build/package/docker-compose.override.yml

```diff
@@ -0,0 +1,98 @@
+version: '3.9'
+
+services:
+  redis:
+    image: redis:6-alpine
+    networks:
+      - walletconnect-monorepo
+    ports:
+      - 6379:6379
+    volumes:
+      - redis:/data
+    healthcheck:
+      test: ["CMD", "redis-cli","ping"]
+      interval: 5s
+      timeout: 5s
+      retries: 5 
+  waku-store:
```

Comment:
@sbc64 I know we have chat about this one and want to check if make sense to remove waku from this setup